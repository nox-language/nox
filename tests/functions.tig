let function maximum(a: int, b: int): int =
        if a > b then
            a
        else
            b
    function minimum(a: int, b: int): int =
        if a < b then
            a
        else
            b
    function sum10(a: int, b: int, c: int, d: int, e: int, f: int, g: int, h: int, i: int, j: int): int =
        a + b + c + d + e + f + g + h + i + j
in (
    printi(minimum(42, 24));
    printi(maximum(42, 24));
    printi(minimum(24, 42));
    printi(maximum(24, 42));
    printi(sum10(1, 2, 3, 4, 5, 6, 7, 8, 9, 10))
)
end
