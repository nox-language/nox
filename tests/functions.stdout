24
42
24
42
55
